buildSrc/src/main/groovy/com/google/android/perftesting/PerfTestTaskGeneratorPlugin.groovy

```diff
@@ -115,19 +118,14 @@ public class PerfTestTaskGeneratorPlugin implements Plugin<Project> {
             process.inputStream.withReader { processOutputReader ->
                 new BufferedReader(processOutputReader).with { bufferedReader ->
                     String outputLine;
+                    // Use regex named groups to check if devices are connected.
+                    // "adb devices -l" to view detail of connected device : <deviceID> device (usb) product model device,
+                    // if use TCPIP to connect debug, you won't see the usb item.
```

Comment:
Comments do not explain why the regex pattern `[\w.:]` is used. How about adding console output for reference?